src/nodes/display/RGBShiftNode.js

```diff
@@ -41,7 +41,7 @@ class RGBShiftNode extends TempNode {
 
 }
 
-export const rgbShift = ( node, amount, angle ) => nodeObject( new RGBShiftNode( nodeObject( node ), amount, angle ) );
+export const rgbShift = ( node, amount, angle ) => nodeObject( new RGBShiftNode( nodeObject( node ).toTexture(), amount, angle ) );
```

Comment:
>  nodeObject( node ).toTexture()

What if the node is already a texture node? Ideally, the conversion is not executed in this casa.